fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/ItemOverlayRendererRegistry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.rendereregistry.v1;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemConvertible;
+
+import net.fabricmc.fabric.impl.client.renderer.registry.ItemOverlayExtensions;
+
+public final class ItemOverlayRendererRegistry {
+	public static ItemOverlayRenderer get(Item item) {
+		return ((ItemOverlayExtensions) item).fabric_getOverlayRenderer();
+	}
+
+	public static void set(Item item, ItemOverlayRenderer overlayRenderer) {
+		((ItemOverlayExtensions) item).fabric_setOverlayRenderer(overlayRenderer);
+	}
+
+	public static ItemOverlayRenderer get(ItemConvertible itemConvertible) {
+		return get(itemConvertible.asItem());
+	}
+
+	public static void set(ItemConvertible itemConvertible, ItemOverlayRenderer overlayRenderer) {
+		set(itemConvertible.asItem(), overlayRenderer);
+	}
+}
```

Comment:
maybe add a private constructor if you don't mind. not critical though

Reply:
should I have it throw an `UnsupportedOperationException` or nah?

Reply:
Just a stub to prevent calls should be fine. after all this is optional, no need to add if you have no other code changes.